### DISCUSSION TITLE
Can't get gpth to run on Synology (Exec format error)

### DISCUSSION THREAD

**Discussion by ElessarDunadan** [TheLastGimbus/GooglePhotosTakeoutHelper#281] [I NEED HELP]
I have downloaded my Google Takeout directly to my Synology NAS, and I want to run gpth on my Synology via SSH.

However, after downloading and "chmod"-ing the gpth-linux file, when I try to run it, I get this error:
`gpth-linux: cannot execute binary file: Exec format error`

I've tried v3.4.2 of gpth-linux (and even gpth-macos), as well as v3.4.1 of gpth-linux, but still keep getting the same result.

Is there something I'm missing?
Has anyone been able to run gpth-linux directly on a Synology NAS for recent versions?
If not, what is the most recent version that ought to work on my NAS?

**Comment by TheLastGimbus** [comment]
i have no idea about this, but visit guys at #157 - you may find answer there

otherwise, google if this isn't a synology-wise problem with many execs
